CityPuzzle/Classes/GameLogic.cs

```diff
@@ -14,14 +15,14 @@ class GameLogic
 
         private double userLat;
         private double userLng;
-        private double questLat;
-        private double questLng;
+        private static double questLat;
```

Comment:
Are you sure these fields can be static. Wont this make quest latitude and longitude unchangeable after the first assignment?